wordpress-mobile/WordPress-iOS
# Only allow published pages in Homepage settings

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/15881

## To test:

From `My Site`:
1. Select `Site Settings`
2. Select `Homepage Settings`
3. Select `Homepage`
    - Make sure `Static Homepage` is selected if you don's see the `Homepage option`
4. **Expect** to see `Choose Homepage` in the title and to only see published pages.

<kbd><a href="https://user-images.githubusercontent.com/3384451/107825092-85879a80-6d50-11eb-9011-7c43b2f9dca4.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/107825092-85879a80-6d50-11eb-9011-7c43b2f9dca4.png"/></a></kbd>

## PR submission checklist:
